oneview/resource_logical_enclosure_test.go

```diff
@@ -0,0 +1,97 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+package oneview
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"testing"
+)
+
+func TestAccLogicalEnclosure_1(t *testing.T) {
+	var logicalEnclosure ov.LogicalEnclosure
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLogicalEnclosureDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccLogicalEnclosure,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckLogicalEnclosureExists(
+						"oneview_logical_enclosure.test", &logicalEnclosure),
+					resource.TestCheckResourceAttr(
+						"oneview_logical_enclosure.test", "name", "Terraform le 1",
+					),
+				),
+			},
```

Comment:
You'll want to add another TestStep to update this resource

Reply:
Added one additional step for update, but not clear on the way how to call this. Not seen any similar code from other resources.

Reply:
This is an array of TestStep so you just have to add another TestStep and change the values to the updated test config and checks
```suggestion
			},
			        Config: testAccLogicalEnclosure,
				Check: resource.ComposeTestCheckFunc(
					testAccCheckLogicalEnclosureExists(
						"oneview_logical_enclosure.test", &logicalEnclosure),
					resource.TestCheckResourceAttr(
						"oneview_logical_enclosure.test", "name", "Terraform le 1",
					),
				),
			{
			}.
```

Reply:
same changes are done